doc/STYLE_GUIDE.md

```diff
@@ -65,6 +65,9 @@
   * Make the "Note:" label italic, i.e. `*Note*:`.
   * Use a capital letter after the "Note:" label.
   * Preferably, make the note a new paragraph for better visual distinction.
+* Function arguments or object properties should use the following format:
+  * <code>* `name` {type|type2} Optional description. **Default:** `defaultValue`</code>
```

Comment:
Should this be rendered or should we escape the special symbols to make them clear? Currently, this is rendered.